desktop-exporter/static/index.html

```diff
@@ -2,11 +2,11 @@
 <html>
 <head>
   <meta charset=utf-8>
-  <title>Title Here</title>
+  <title>Otel Desktop Viewer</title>
```

Comment:
Should we use "OpenTelemetry" here instead of "Otel"?

Reply:
Yes.